### DISCUSSION TITLE
How to set temporary directory in Perl Image::Magick

### DISCUSSION THREAD

**Discussion by fkoyer** [ImageMagick/ImageMagick#7082] [General]
I can change the directory used by ImageMagick to create temporary files from the command line using

```
convert -define registry:temporary-path=/my/temp/dir ...
```

How do I do it in Perl using Image::Magick?

**Comment by urban-warrior** [comment]
Grab the latest Github main and use this statement:

```
$q->Set("registry:temporary-path","/my/temp/dir");
```
